src/domain/app/layout/mainContent.module.scss

```diff
@@ -0,0 +1,6 @@
+.mainContent::before {
+  content: '';
+  display: block;
+  height: 130px;
+  margin-top: -130px;
+}
```

Comment:
The purpose of these styles is to fix scroll position when user is clicking Skip to content link. Otherwise the browser scrolls to the beginning of main content and part of the content is hidden behind header.

<img width="1362" alt="Screenshot 2021-01-14 at 14 15 50" src="https://user-images.githubusercontent.com/24706814/104596158-4fd19200-567c-11eb-9dea-0e1a346c6446.png">
VS
<img width="1440" alt="Screenshot 2021-01-14 at 14 11 54" src="https://user-images.githubusercontent.com/24706814/104596161-519b5580-567c-11eb-92d3-874fb0518328.png">